README.md

```diff
@@ -17,7 +17,7 @@ Reference implementation of the zigpy library exist in **[Home Assistant](https:
 
 zigpy have code to automatically download and perform OTA (Over-The-Air) firmware updates of Zigbee devices if the OTA firmware image provider source URL for updates is known. The OTA update directory is optional and it can also be used for any offline firmware files that you provide yourself.
 
-Online OTA providers for firmware updates are currently only available for IKEA, LEDVANCE and SALUS devices. Support for OTA updates from other manufacturers could be added to zigpy in the future, if they publish their firmware images publicly.
```

Comment:
If we want to remove the capitalisation (do we?) we should do it on all vendors.

Reply:
No, it depends on if the company name. IKEA company name for example is actually an abbreviation so should always be written in capital letters. LEDVANCE company is also written in capital letters. So maybe just quickly check wikipedia https://en.wikipedia.org/wiki/IKEA https://en.wikipedia.org/wiki/LEDVANCE https://en.wikipedia.org/wiki/Salus

Reply:
@hedda You are right.  I thought IKEA, LEDVANCE were just capitalised by coder choice.  But actually all apart from Computime capitalise their names in the brand.
So the PR with the latest commit is fine by me now.

Reply:
I guess then the only nit-pick now is if they should be listed as "SALUS/Computime" or "Computime/SALUS"? :P